### PR TITLE
Nicer graphs for testsuites

### PR DESCRIPTION
**Pull request by jonbell-lot23**
Old:
image
image

New:
image
image

**Some notes**

We have a few Safari bugs that are emerging. Before this PR, the chart order is reversed on Safari. (https://linear.app/replay/issue/SCS-1782/our-graph-is-reversed)

And as part of this PR, the colors don't work right on Safari. I'll look into this in another PR, since Safari marketshare is pretty much just me.

### CODE REVIEW

**Comment by jonbell-lot23** [comment]
Addresses https://linear.app/replay/issue/DES-1079/improve-our-testsuites-graph

File: src/ui/components/Library/Team/View/NewTestRuns/TestRunsStats copy.tsx
Comment (ryanjduffy) [inline src/ui/components/Library/Team/View/NewTestRuns/TestRunsStats copy.tsx:1]: I assume this was an accidental commit

**Review by ryanjduffy — APPROVED** [review]
One comment but :+1: otherwise

File: src/ui/components/Library/Team/View/NewTestRuns/TestRunsStats copy.tsx
Comment (jonbell-lot23) [reply src/ui/components/Library/Team/View/NewTestRuns/TestRunsStats copy.tsx:1]: Oops

**Pull request merged**